include/erlang_ls.hrl

```diff
@@ -579,6 +579,7 @@
                    | include_lib
                    | macro
                    | module
+                   | parse_transform
```

Comment:
This was already  available as a `poi_kind()` but missing in the type definition. The new code highlighted the issue.